index.html

```diff
@@ -36,6 +36,40 @@ <h2>Donating To Us</h2>
 		<div class="content-divider">
 		</div> <!-- End of content-divider -->
 
+		<div class="card-component-container">
+			<h2>More Ways To Help</h2>
+			<div class="row-container">
+				<div class="card-wrapper">
+					<div class="card card-one">
+						<a href="#"><img src="img/volunteer.jpeg" alt="volunteers" class="illustrative-image"></a>
```

Comment:
Give classes to anchor tags and style them. `width: 100%` and `height: 225px` will look good.

Reply:
Perhaps update illustrative-image class to "card-image" to be constant? Class in css would also need updating.

Reply:
done